isaacConfig.cmake

```diff
@@ -0,0 +1,141 @@
+# - Config file for the isaac package
+# It defines the following variables
+#  isaac_INCLUDE_DIRS - include directories for FooBar
+#  isaac_LIBRARIES    - libraries to link against
+#  isaac_DEFINITIONS  - necessary definitions
+#
+# It defines the following options
+#  ISAAC_THREADING
+#  ISAAC_SHOWBORDER
+#  ISAAC_CUDA
+#  ISAAC_ALPAKA
+#  ISAAC_JPEG
+
+###############################################################################
+# isaac
+###############################################################################
+cmake_minimum_required (VERSION 2.6)
+cmake_policy(SET CMP0048 OLD)
+project (isaac)
+
+set(isaac_INCLUDE_DIRS ${isaac_INCLUDE_DIRS} "${isaac_DIR}/insitu")
+set(isaac_INCLUDE_DIRS ${isaac_INCLUDE_DIRS} "${isaac_DIR}/insitu/isaac")
+
+set(isaac_DEFINITIONS ${isaac_DEFINITIONS} "-std=c++11")
+set(isaac_DEFINITIONS ${isaac_DEFINITIONS} "-march=native")
+set(isaac_DEFINITIONS ${isaac_DEFINITIONS} "-mtune=native")
+set(isaac_DEFINITIONS ${isaac_DEFINITIONS} "-DISAAC_MAX_FUNCTORS=3")
+set(isaac_DEFINITIONS ${isaac_DEFINITIONS} "-DISAAC_FUNCTOR_POW_ENABLED=0")
+
+###############################################################################
+# MODULES
+###############################################################################
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${isaac_DIR}/Modules")
+
+
+###############################################################################
+# OPTIONS
+###############################################################################
+option(ISAAC_THREADING "Do the video and metadata transmission transport in background. May be buggy, but seems to work fine!" OFF)
```

Comment:
but afaik the official spelling is `ISAAC` so both environment options (always uppercase) and cmake vars (original spelling _UPERCASEOPTION) are the same writing